src/pymor/reductors/neural_network.py

```diff
@@ -423,13 +495,19 @@ class NeuralNetworkStatefreeOutputReductor(NeuralNetworkReductor):
         The full-order |Model| to reduce. If `None`, the `training_set` has
         to consist of pairs of |parameter values| and corresponding outputs.
     training_set
-        Set of |parameter values| to use for POD and training of the
-        neural network. If `fom` is `None`, the `training_set` has
-        to consist of pairs of |parameter values| and corresponding outputs.
+        List of |Parameter values| to use for POD and training of the
+        neural network.
+    training_snapshots
+        Set of solution |VectorArrays| to use for POD and training of the
```

Comment:
```suggestion
        Set of solution |VectorArrays| to use for training of the
```

Reply:
Is the user supposed to pass high-dimensional solutions here? That would definitely seem wrong to me...